cookbooks/Berksfile

```diff
@@ -6,4 +6,7 @@ cookbook 'apt', '~> 2.3.8'
 cookbook 'build-essential', '~> 2.1.3'
 cookbook 'chef-client', '~> 4.2.0'
 cookbook 'ntp', '~> 1.8.6'
-cookbook 'brightbox-ruby', '~> 1.2.1'
+
+Dir.entries('.').select{|x| File.directory?(x) && !x.start_with?('.')}.each do |path|
+  cookbook(File.basename(path), path: path)
```

Comment:
It wouldn't be a bad idea to explicitly list all of our cookbooks in the Berksfile if necessary, but I'd rather avoid needing to keep it manually maintained if possible, since it would be another thing we'd need to document/remember to update, and another potential place for human-error bugs to pop up.

There are less fragile ways of getting this list- I'll commit an approach that uses `knife cookbook list -z` to populate the list which should use the same logic Chef does to list cookbooks on the chef-repo.